docs/source/troubleshooting.rst

```diff
@@ -0,0 +1,35 @@
+Troubleshooting
+===============
+The following sections provide a reference for viewing logging information from the command line and server to enable troubleshooting of various activities being performed with Quipucords.
+
+Troubleshooting the Command Line
+--------------------------------
+Logs for the command line can be found in the ``~/.local/share/qpc/`` directory with the name **qpc.log**. Logging information is captured for each command with the input arguments and request endpoint and response information to enable situational debugging. The command line supports a global flag ``-v`` to increase verbosity of the logs.
+
+Troubleshooting the Server
+--------------------------
+Logs from the server are mapped from the running container into a users specified directory. If you used the defaults found in `Selecting the Quipucords Server Configuration Options <configure.html#selecting-the-quipucords-server-configuration-options>`_ section, the directory would be ``~/quipucords/log``. You will find three log files within this directory:
```

Comment:
If you used the defaults found in `link` section --> If you used the defaults found in the `link` section

Reply:
done